vru_use_case_scripts/extract_rtf_data

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sed -e "s/\r/\n/g" \
```

Comment:
Do we want to write column headers/labels

Reply:
I think we can leave off column headers because it will affect the downstream `plot_rtf_data` script. Robustly determining if CSV data has a header seems deceptively difficult.

Reply:
Addressed